### DISCUSSION TITLE
Como apontar que uma variável é NOT NULL no VS Code

### DISCUSSION THREAD

**Discussion by guifsqueiroz** [balta-io/forum#58] [Tira Dúvidas] [answered]
Boa noite, quando eu crio as propriedades do banco de dados no VSCode, ele aponta que eu deveria declarar a variável como não nula, uma vez que ela é NOT NULL no banco de dados, tentei a sugestão do editor que seria escrever algo como `public string? Name { get; set; }` Porém dessa forma foi possível fazer insert's de dados vazios no banco. Como eu posso corrigir esse aviso?
![vs_eror](https://user-images.githubusercontent.com/112831458/196822401-b97aa3d2-df3d-411e-b56a-d609134dbfc1.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @guifsqueiroz, como vai?

Este aviso é do .NET 6 (6 em diante) que diz que uma variável que pode ser nula não foi inicializada ou declarada como nullable. Isto é algo do .NET e não está relacionado ao seu banco de dados.

Para resolver este problema, basta inicializar a variável:
```csharp
public string Name { get; set; } = string.Empty;
```

**Reply by guifsqueiroz** [reply]
Deu certo, vlw professor!
